owasp-security-logging-logback/pom.xml

```diff
@@ -30,6 +30,9 @@
     <developerConnection>scm:git:git@github.com:javabeanz/owasp-security-logging.git</developerConnection>
     <url>https://github.com/javabeanz/owasp-security-logging</url>
   </scm>
+  <properties>
```

Comment:
I followed the pattern that was created for the log4j repo and created a property at the top level.

Reply:
Thanks. I was working on the exact same thing but I'll happily use yours.

Reply:
thank you @augustd !